content/en/vehicle-management-computer/preparing-the-jetson/_index.md

```diff
@@ -26,5 +26,11 @@ description: "Physically assemble the Jetson"
 ![](DSC02233.jpg)
 
 {{% alert title="Success" color="success" %}}
-Now you're ready to flash the SD Card!
+Now that your Jetson assembly is complete, please be sure to insert the micro SD card. The micro SD card contains Nvidia's operating sytem and the default VMC software already installed.
```

Comment:
The micro SD card contains *Ubuntu or *Linux instead of "Nvidia's operating system"